bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "px-vis",
-  "version": "5.1.6",
+  "version": "5.1.7",
```

Comment:
This will actually be 5.1.8 (my previous PR to bump the bower.json version was never merged). 

We should also make the same change to package.json version. (to 5.1.8)